src/Generators/DataTablesMakeCommand.php

```diff
@@ -128,6 +207,8 @@ protected function getOptions()
     {
         return [
             ['model', null, InputOption::VALUE_NONE, 'Use the provided name as the model.', null],
+            ['action', null, InputOption::VALUE_OPTIONAL, 'Force the use of singular in filename.', null],
```

Comment:
Oops, i've totally messed this, it's a copy/paste of an other package, but i've forgotten to change the description